.circleci/config.yml

```
@@ -838,8 +856,16 @@ jobs:
       - run:
           name: strip binary
           command: strip build/solc/solc
-      - store_artifacts: *artifacts_solc
-      - persist_to_workspace: *artifacts_executables
+      - store_artifacts:
+          path: build/solc/solc
+          destination: solc-static-linux
+      - run: |
+            mkdir -p static-build/solc
+            cp build/solc/solc static-build/solc/solc-static-linux
```

Why not rename just the executable?

```suggestion
      - run: mv build/solc/solc build/solc/solc-static-linux
```

---

Indeed.